http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/app/WdsAppInstall.scala

```diff
@@ -95,6 +98,7 @@ class WdsAppInstall[F[_]](config: WdsAppConfig,
           // general configs
           raw"fullnameOverride=wds-${params.app.release.asString}",
           raw"instrumentationEnabled=${config.instrumentationEnabled}",
+          raw"vpaEnabled=${vpaEnabled}",
```

Comment:
nit: instead of a top-level "vpaEnabled" value, this should probably go up at line ~74 with the other "azure resources configs" and get renamed to "config.vpaEnabled" or better yet "config.aks.vpaEnabled"

Reply:
Thanks, I wasn't sure where to put it and wasn't sure whether to start a whole aks category